gremlin-server/src/test/java/org/apache/tinkerpop/gremlin/driver/remote/GraphBinaryGroovyRemoteGraphProvider.java

```diff
@@ -16,23 +16,20 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.tinkerpop.gremlin.jsr223;
+package org.apache.tinkerpop.gremlin.driver.remote;
 
-import org.apache.tinkerpop.gremlin.process.traversal.GremlinLang;
-import org.apache.tinkerpop.gremlin.process.traversal.Translator;
+import org.apache.tinkerpop.gremlin.structure.Graph;
+import org.apache.tinkerpop.gremlin.util.ser.SerializersV4;
 
 /**
- * Provides a way to customize and override {@link GremlinLang} to script translation. Not all {@link GremlinScriptEngine}
- * will support this capability as translation is optional.
- *
  * @author Stephen Mallette (http://stephen.genoprime.com)
  */
-public interface TranslatorCustomizer extends Customizer {
-
-    /**
-     * Construct a {@link Translator.ScriptTranslator.TypeTranslator} that will be used by a
-     * {@link Translator.ScriptTranslator} instance within the {@link GremlinScriptEngine} to translate
-     * {@link GremlinLang} to a script.
-     */
-    public Translator.ScriptTranslator.TypeTranslator createTypeTranslator();
+@Graph.OptOut(
```

Comment:
do all the tests need to opted out? only the `GreedyMatchTraversals` use that strategy syntax, no?